DateTimeForm/index.jsx

```diff
@@ -77,6 +80,8 @@ DateTimeForm.propTypes = {
     month: PropTypes.number,
     year: PropTypes.number,
   }),
+  select24Hours: PropTypes.bool,
+  firstDayOfWeek: InputDate.defaultProps.firstDayOfWeek,
 };
```

Comment:
Super simple and feels really clean and consistent with how we handle `select24Hours` props in other components! 👍